README.md

```diff
@@ -298,6 +306,24 @@ console.log("will raise an exception if not in script_hashes.yml!")
 <% end %>
 ```
 
+### Public Key Pins
+
+```
+config.hpkp = {
+  max_age: 60.days.to_i,   # max_age is a required parameter
+  include_subdomains: true, # whether or not to apply pins to subdomains
+  # Per the spec, SHA256 hashes are the only currently supported format.
+  pins: [
```

Comment:
LOL `secure_headers` thrives on awkward abuse of the `Hash` class. 

But seriously, I like the "array of hashes" syntax more than either suggestion. I never said I was good at API design.